package.json

```diff
@@ -71,31 +69,34 @@
     "import-sort-style-eslint": "^4.2.0",
     "json-stable-stringify": "^1.0.1",
     "lodash": "^4.17.4",
+    "set-abi-gen": "1.1.0-beta.1",
     "solc": "0.4.24",
     "solidity-coverage": "^0.5.5",
     "solidity-sha3": "^0.4.1",
-    "truffle": "4.1.11",
+    "truffle": "5.0.0-beta.1",
     "tslint": "^5.8.0",
     "tslint-no-unused-expression-chai": "0.0.3",
     "types-bn": "^0.0.1",
     "types-ethereumjs-util": "^0.0.5",
-    "typescript": "^2.9.2",
-    "web3": "0.20.0"
+    "typescript": "^3.1.2"
   },
   "dependencies": {
+    "bn-chai": "^1.0.1",
     "cryptofin-solidity": "^0.0.5",
     "eth-gas-reporter": "^0.1.10",
+    "ethers": "^4.0.5",
     "fs-extra": "^5.0.0",
     "husky": "^0.14.3",
     "lint-staged": "^7.2.0",
     "module-alias": "^2.1.0",
-    "set-protocol-utils": "^0.3.37",
+    "set-protocol-utils": "^1.0.0-beta.10",
     "sol-trace-set": "^0.0.1",
     "solium": "^1.1.7",
     "tiny-promisify": "^1.0.0",
-    "truffle-hdwallet-provider": "^0.0.3",
+    "truffle-hdwallet-provider": "^1.0.0-web3one.0",
```

Comment:
That's cool this dep has a web3 1.0 version

Reply:
it is required as part of the truffle 5 migration efforts too i think